### ISSUE TITLE
Identity insert error

### ISSUE THREAD

**Issue by ger2902** [sqlite-sync/SQLite-sync.com#53]
Hi, I'm getting this error when I try to insert in the master db a new record that has an Identity primary key:

ERROR PushInsertRecords() Cannot insert explicit value for identity column in table 'Table' when IDENTITY_INSERT is set to OFF.

is an special configuration required??

Thanks

**Comment by sqlite-sync** [comment]
Check if you have IDENTITY property on primary key set to TRUE.

**Comment by ger2902** [comment]
Yes it's set to TRUE, is it possible to insert new records with identity value? I wonder if should I send as null the value in the identity column...
